src/main/java/weavers/siltarae/global/exception/ExceptionCode.java

```diff
@@ -11,6 +11,9 @@ public enum ExceptionCode {
     INVALID_MISTAKE_CONTENT_SIZE(2001, "실수 내용은 140자가 넘을 수 없습니다."),
     INVALID_MISTAKE_CONTENT_NULL(2002, "실수 내용은 필수입니다."),
     INVALID_MISTAKE_CONTENT(2003, "실수 내용에는 한글, 영어, 숫자만 허용됩니다."),
+    NOT_FOUND_MISTAKE(2004, "해당하는 실수가 존재하지 않습니다."),
+
+    NOT_FOUND_USER(3001, "해당하는 유저가 존재하지 않습니다."),
```

Comment:
3001번 코드는 어제 제가 `DUPLICATED_TAG_NAME`에 할당했던 것 같아서요..! 충돌되지 않게 4000번대는 어떨까요 👀 

```suggestion
    NOT_FOUND_USER(4001, "해당하는 유저가 존재하지 않습니다."),
```

Reply:
좋아요!!